refresh.template.py

```diff
@@ -414,6 +454,8 @@ def _get_commands(target: str, flags: str):
         # Shush logging. Just for readability.
         '--ui_event_filters=-info',
         '--noshow_progress',
+        # Disable param file, only useful for Windows. No effect on Linux.
```

Comment:
Could we document here why we're doing this? eg Not needed for aquery because command is not run, and that we want the command even if the code doesn't build

Reply:
Sure